### DISCUSSION TITLE
Google auth only works for superadmins

### DISCUSSION THREAD

**Discussion by ghubz** [WeblateOrg/weblate#7108] [Q&A]
Hi,

On my installation of weblate Google Auth only works for superadmins. All other users get an error message that Registration is disabled.

Any ideas?

Thanks

**Comment by nijel** [comment]
Perhaps you have [new registrations disabled](https://docs.weblate.org/en/latest/admin/config.html#registration-open)?

**Reply by ghubz** [reply]
I do, but I want existing users to be able to login  with oauth. Is that not possible?
To have existing users login but disable new registration I must add google to REGISTRATION_ALLOW_BACKENDS  ?

**Reply by nijel** [reply]
When registration is enabled, users should be able to associate Google with their existing account once they are logged in.
